src/test/java/uk/gov/companieshouse/api/accounts/validation/StocksValidatorTest.java

```diff
@@ -33,6 +25,16 @@
 import uk.gov.companieshouse.api.accounts.service.response.ResponseStatus;
 import uk.gov.companieshouse.api.model.transaction.Transaction;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
 @ExtendWith(MockitoExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class StocksValidatorTest {
```

Comment:
Can we add a unit test to cover `getAccountingNoteType` for this class and the transformer please?